inc/lift/Request.h

```diff
@@ -349,6 +433,45 @@ class Request {
     /// libuv will call this function when the AddRequest() function is called.
     friend auto requests_accept_async(
         uv_async_t* handle) -> void;
+    
+    friend auto on_response_wait_time_expired_callback(uv_timer_t* handle) -> void;
+};
+
+/**
+ * Class wrapping information used by EventLoop to "time out" requests that have not
+ * received a response within the expected wait time.
+ */
+class ResponseWaitTimeWrapper
```

Comment:
Seems like this could be a simple struct -- can use `const` to make it read-only

Reply:
Because this class's main job is to ensure correct order in the set of requests that will be checked for response wait time, I think it makes sense to keep the operator as part of the class.